src/DependencyInjection/DoctrineEncryptExtension.php

```diff
@@ -58,9 +59,21 @@ public function load(array $configs, ContainerBuilder $container): void
             $loader->load('services_with_secret.yml');
         }
 
-        // Remove usage of AttributeAnnotationReader when using php < 8.0
-        if (PHP_VERSION_ID < 80000) {
-            $container->setAlias('ambta_doctrine_annotation_reader','annotations.reader');
+        // Symfony 1-3
+        if (Kernel::MAJOR_VERSION < 5) {
+            throw new \RuntimeException('doctrineencryptbundle/doctrine-encrypt-bundle expects symfony-version >= 4.1!');
+        // Symfony 4-6
+        } elseif (Kernel::MAJOR_VERSION < 7) {
+            // PHP 7.x (no attributes)
+            if (PHP_VERSION_ID < 80000) {
+                $loader->load('services_subscriber_sf4_php7.yml');
```

Comment:
Perhaps we should rename these files to 
* `services_subscriber_with_annotations.yml`
* `services_subscriber_with_annotations_and_attributes.yml`
* `service_listeners_with_attributes.yml`